README.md

```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+The SIG creates programs and processes and consults other SIGs to improve the overall community health. 
+
+## Get Involved
+
+There are many ways to get involved in our work:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls)
+- Get in touch with us on the [CNCF slack](https://slack.cncf.io/) channel [#otel-contributor-experience](https://cloud-native.slack.com/archives/C06TMJ2R0SK)
+- [Attend one of our meetings](https://docs.google.com/document/d/1CTQI0p3QF8JP8reV8z_ggcs8KE5YVPpQGvAQknw4qP0/edit#heading=h.x3xl5syz15zx), we alternate between two times to make it possible for you to join no matter which time zone you live or work in! 
+
+> [!NOTE]
+> This SIG operates **async** by default, attending any meetings is not mandatory! Feel free to choose your preferred way of communication
```

Comment:
```suggestion
> This SIG operates **async** by default, attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

Reply:
```suggestion
> This SIG operates **async** by default, so attending meetings is not mandatory! Feel free to choose your preferred way of communication.
```

Reply:
merging @tiffany76's and @kaylareopelle's suggestions